bundle/kubernetes/manifests/nsx-container-plugin-operator.clusterserviceversion.yaml

```diff
@@ -101,7 +101,7 @@ spec:
                   value: nsx-ncp:latest
                 - name: WATCH_NAMESPACE
                   value: nsx-system-operator
-                image: docker.io/vmware/nsx-container-plugin-operator:v3.2.0
+                image: docker.io/vmware/nsx-container-plugin-operator
```

Comment:
It looks like here should be : docker.io/vmware/nsx-container-plugin-operator:v3.2.1? not sure how to generate it or edit version manually? 
The same question in code line #L22
name: nsx-container-plugin-operator.v3.2.0

Reply:
This a bit of a chicken and egg problem, since the tagged image is generated upon release.
I'd say let's keep it untagged in master, and then we apply the tag on the release branch

Reply:
Hi Salvatore, Thanks for explaination. This patch will be merged into master, so I think that other code with v3.2.0 tag should be corrected  without tag as well, is it?  such as: 
code line #L11
containerImage: vmware/nsx-container-plugin-operator:3.2.0
code line #L22
name: nsx-container-plugin-operator.v3.2.0

I'm planning to cut release_v3.2.1 branch upon the PR merged, and then change image with tag v3.2.1.

Reply:
That's correct. Your change does the right thing in my opinion

Reply:
Hi Salvatore, if changes go well, could you help to approve this PR, Thanks a lot.